src/Draco.Editor.Web/Program.cs

```diff
@@ -1,14 +1,176 @@
-using Microsoft.AspNetCore.Components.Web;
+using System.IO.Compression;
+using System.Text;
+using Draco.Compiler.Api.Scripting;
+using ICSharpCode.Decompiler.Disassembler;
+using ICSharpCode.Decompiler.Metadata;
+using ICSharpCode.Decompiler;
+using Microsoft.AspNetCore.Components;
 using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
+using Microsoft.JSInterop;
 
 namespace Draco.Editor.Web;
 
 public class Program
 {
+    private static IJSRuntime js = null!;
+    private static IJSObjectReference appJS = null!;
+    private static string code = "";
     public static async Task Main(string[] args)
```

Comment:
Please separate method and fields with a blank line

Reply:
![image](https://user-images.githubusercontent.com/18743295/201440563-8a62d849-08b0-442f-9d2a-be9bd1debd5e.png)
I'd argue codelense do it :p